packages/datx/src/helpers/model/fields.ts

```diff
@@ -41,6 +41,14 @@ export function updateRef(
 ): PureModel | Array<PureModel> | null {
   const bucket: IBucket<PureModel> | undefined = getMeta(model, `ref_${key}`);
 
+  if (isIdentifier(value) || Array.isArray(value)) {
```

Comment:
It would be safer to use the mobx `isArrayLike` method here, just to also cover the case where this value is an observable array.